UnitsNet/UnitDefinitions/Duration.json

```diff
@@ -162,6 +162,18 @@
           "Abbreviations": [ "нс" ]
         }
       ]
+    },
+    {
+      "SingularName": "Cycle",
```

Comment:
Small suggestion toward the above code snippet. I would use the .As() method on Frequency instead of that switch statement, since its more readable.

```
Frequency sourceFrequency;
if (Frequency.TryParse(sourceQuantity, out sourceFrequency))
{
  FrequencyUnit targetFrequencyUnit;
  if (FrequencyWrapper.TryParseUnit(targetUom, out targetFrequencyUnit))
  {
    result = sourceFrequency.As(targetFrequencyUnit);
    return true;
  }
}
```

Feel free to ignore this suggestion, of course.